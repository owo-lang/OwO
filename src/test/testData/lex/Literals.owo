123 "dio!" "ayaya" "123"
"\n" '\n' "a\n"
"\"" "\'" "'"
'|' '"' abc
