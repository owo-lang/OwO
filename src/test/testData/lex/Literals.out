IntegerToken 123 (0 1 1) (3 1 4)
StringToken "dio!" (4 1 5) (10 1 11)
StringToken "ayaya" (11 1 12) (18 1 19)
StringToken "123" (19 1 20) (24 1 25)
StringToken "\n" (25 2 1) (29 2 5)
CharToken '\n' (30 2 6) (34 2 10)
StringToken "a\n" (35 2 11) (40 2 16)
StringToken "\"" (41 3 1) (45 3 5)
StringToken "'" (46 3 6) (50 3 10)
StringToken "'" (51 3 11) (54 3 14)
CharToken '|' (55 4 1) (58 4 4)
CharToken '"' (59 4 5) (62 4 8)
IdentifierToken "abc" (63 4 9) (66 4 12)
